app/blog/2018-01-25-polymer-2-4.md

```diff
@@ -0,0 +1,90 @@
+
+`my-element.html`
+```html
+<link rel="import" href="bower_components/polymer/polymer-element.html>
+<dom-module id="my-element"><template>...</template></dom-module>
+<script src="my-element.js">
+```
+
+`my-element.ts`
+```ts
+  /// <reference path="./bower_components/polymer/types/polymer-element.d.ts" />`
+  class MyElement extends Polymer.Element {
+    ...
+  }
+```
+
+We’re also introducing [`polymer-decorators`](https://github.com/Polymer/polymer-decorators), an officially supported set of decorators to help you write Polymer elements with TypeScript in a type-safe, convenient way. This helps TypeScript understand the shape of your elements, giving you code completion and type checking on element properties.
+
+In the coming months, expect to see additional support for TypeScript, including typings for Polymer 3.0, typings for the elements in the Polymer catalog, and support for analyzing TypeScript projects so that they can be used with `<iron-doc-viewer>` and other tools.
+
+## `PropertiesMixin`
+
+In Polymer 2.4 we have broken out a key subset of Polymer’s lower-level properties functionality into a tiny new mixin called `PropertiesMixin`. This mixin implements Polymer’s features for defining declarative properties, creating property accessors, and syncing properties with attributes. It provides a single `_propertiesChanged` callback and can be used to create new, lightweight base classes. 
+
+To illustrate, the following example shows a very simple element using `PropertiesMixin`:
+
+```html
+<link rel="import" href="bower_components/polymer/lib/mixins/properties-mixin.html">
+
+<script>
+  class MyPropertiesElement extends Polymer.PropertiesMixin(HTMLElement) {
+    // Define properties to watch. You may only specify the property’s name and type.
+    static get properties() {
+      return {
+        name: String
+      }
+    }
+    // Called whenever the declared properties change. 
+    _propertiesChanged(currentProps, changedProps, oldProps) {
+      // Render the changed content.
+      this.textContent = `Hello, ${this.name}`;
+    }
+  }
+</script>
+```
+
+This is an intentionally simple example; we don't expect many people to use `PropertiesMixin` directly. It's intended to provide support for new base classes.  For more information, see the source for `PropertiesMixin`.
+
+## `Polymer.html` tag function
+
+Starting with this release, you need to return an instance of `HTMLTemplateElement` from your element’s static template getter—the option to return a string has been deprecated. This change makes the template type predictable, which in turn will make it simpler to extend superclass templates.
+
+If–like most Polymer developers–you are using `<dom-module>` to define your templates in HTML, you don't need to make any changes.  Your templates will continue to work in 2.4 (and any subsequent 2.x releases) as they always have, and Polymer Modulizer will automatically convert your templates when you update to Polymer 3.0.
```

Comment:
I would bold **you don't need to make any changes.** Most important phrase in this entire section.